health/health.d/disks.conf

```diff
@@ -150,7 +150,7 @@ component: Disk
      warn: $this > $green * (($status >= $WARNING)  ? (0.7) : (1))
      crit: $this > $red   * (($status == $CRITICAL) ? (0.7) : (1))
     delay: down 15m multiplier 1.2 max 1h
-     info: average percentage of time the disk was busy over the last 10 minutes
+     info: average percentage of time $family disk was busy over the last 10 minutes
        to: silent
```

Comment:
Hi @ilyam8 , is this one supposed to have `to: silent` by default?

Reply:
yes

Reply:
> If this is 100%, it doesn’t always mean that the device is saturated. Depends on a number of oustanding requests and the number of concurrent requests the device can handle. For instance, if the device is able to handle 8 concurrent requests and there are 2 we would see 100% untilization, but there is no problem at all.

from https://github.com/netdata/netdata/pull/10688

Reply:
Many thanks man!